CHANGELOG.md

```diff
@@ -1,24 +1,1031 @@
+  - add Code.Tag method
+  - add the KnownCodes API
+  - use "go run pkg@version" assuming Go 1.17 or later
+  - update submodule and re-generate
+  - update to newer multicodec table ([multiformats/go-multicodec#57](https://github.com/multiformats/go-multicodec/pull/57))
+  - Update `multicodec` submodule to `1bcdc08` for CARv2 index codec
+- github.com/multiformats/go-multistream (v0.2.2 -> v0.3.0):
+  - release v0.3.0 (#82) ([multiformats/go-multistream#82](https://github.com/multiformats/go-multistream/pull/82))
+  - catch panics (#81) ([multiformats/go-multistream#81](https://github.com/multiformats/go-multistream/pull/81))
+  - sync: update CI config files (#78) ([multiformats/go-multistream#78](https://github.com/multiformats/go-multistream/pull/78))
+  - reduce the maximum read buffer size from 64 to 1 kB ([multiformats/go-multistream#77](https://github.com/multiformats/go-multistream/pull/77))
+  - remove unused ls command ([multiformats/go-multistream#76](https://github.com/multiformats/go-multistream/pull/76))
+  - chore: remove empty file cases.md ([multiformats/go-multistream#75](https://github.com/multiformats/go-multistream/pull/75))
+  - chore: remove .gx ([multiformats/go-multistream#72](https://github.com/multiformats/go-multistream/pull/72))
+  - don't commit the fuzzing binary ([multiformats/go-multistream#74](https://github.com/multiformats/go-multistream/pull/74))
+  - sync: update CI config files (#71) ([multiformats/go-multistream#71](https://github.com/multiformats/go-multistream/pull/71))
+  - remove Makefile ([multiformats/go-multistream#67](https://github.com/multiformats/go-multistream/pull/67))
+
+</details>
+
+### ❤ Contributors
+
+| Contributor | Commits | Lines ± | Files Changed |
+|-------------|---------|---------|---------------|
+| Jorropo | 16 | +79107/-72184 | 25 |
+| Marten Seemann | 367 | +15210/-12905 | 882 |
+| vyzo | 135 | +7953/-1774 | 234 |
+| Will Scott | 36 | +2772/-1783 | 90 |
+| Daniel Martí | 33 | +2993/-862 | 89 |
+| Will | 21 | +3004/-354 | 95 |
+| Steven Allen | 88 | +2474/-868 | 136 |
+| Marcin Rataj | 25 | +2249/-804 | 61 |
+| Masih H. Derkani | 14 | +2068/-861 | 71 |
+| Adin Schmahmann | 34 | +2264/-484 | 75 |
+| Gus Eggert | 20 | +2102/-590 | 87 |
+| Lucas Molas | 24 | +1288/-520 | 70 |
+| Raúl Kripalani | 18 | +1519/-271 | 38 |
+| Rod Vagg | 18 | +1392/-395 | 36 |
+| Brian Tiger Chow | 20 | +833/-379 | 40 |
+| Jeromy Johnson | 53 | +646/-302 | 83 |
+| Łukasz Magiera | 26 | +592/-245 | 43 |
+| Artem Mikheev | 2 | +616/-120 | 5 |
+| Franky W | 2 | +49/-525 | 9 |
+| Laurent Senta | 3 | +468/-82 | 52 |
+| Juan Batiz-Benet | 8 | +285/-80 | 18 |
+| Hector Sanjuan | 26 | +206/-154 | 55 |
+| Justin Johnson | 2 | +181/-88 | 2 |
+| Thibault Meunier | 5 | +216/-28 | 8 |
+| Aayush Rajasekaran | 2 | +133/-103 | 11 |
+| James Wetter | 2 | +234/-1 | 2 |
+| gammazero | 7 | +140/-84 | 12 |
```

Comment:
```suggestion
| Marten Seemann | 367 | +15210/-12905 | 882 |
| vyzo | 135 | +7953/-1774 | 234 |
| Will Scott | 36 | +2772/-1783 | 90 |
| Daniel Martí | 33 | +2993/-862 | 89 |
| Will | 21 | +3004/-354 | 95 |
| Steven Allen | 88 | +2474/-868 | 136 |
| Marcin Rataj | 25 | +2249/-804 | 61 |
| Masih H. Derkani | 14 | +2068/-861 | 71 |
| Adin Schmahmann | 34 | +2264/-484 | 75 |
| Gus Eggert | 20 | +2102/-590 | 87 |
| Lucas Molas | 24 | +1288/-520 | 70 |
| Raúl Kripalani | 18 | +1519/-271 | 38 |
| Rod Vagg | 18 | +1392/-395 | 36 |
| Brian Tiger Chow | 20 | +833/-379 | 40 |
| Jeromy Johnson | 53 | +646/-302 | 83 |
| Łukasz Magiera | 26 | +592/-245 | 43 |
| Artem Mikheev | 2 | +616/-120 | 5 |
| Franky W | 2 | +49/-525 | 9 |
| Laurent Senta | 3 | +468/-82 | 52 |
| Juan Batiz-Benet | 8 | +285/-80 | 18 |
| Hector Sanjuan | 26 | +206/-154 | 55 |
| Justin Johnson | 2 | +181/-88 | 2 |
| Thibault Meunier | 5 | +216/-28 | 8 |
| Aayush Rajasekaran | 2 | +133/-103 | 11 |
| James Wetter | 2 | +234/-1 | 2 |
| gammazero | 7 | +140/-84 | 12 |
| Jorropo | 16 | +97/-96 | 25 |
```
This removes the outlier file and put me down in the list where I should be (assuming it's sorted by `abs(adds) + abs(remove)`).

Reply:
Turned out doing the easy way took like 2 seconds (see the `mkreleaselog` file change I made)

Reply:
Really good